pkg/apis/machine/types.go

```diff
@@ -92,6 +95,27 @@ type MachineTemplateSpec struct {
 	Spec MachineSpec
 }
 
+// MachineControllerConfig describes the configurations useful for the machine-controller.
+type MachineControllerConfig struct {
```

Comment:
Is it better that we call this `MachineConfiguration`? Not sure just was thinking of a better name

Reply:
Yes, I am fine with either. Let me know if it should be changed to MachineConfiguration.

Reply:
Anything is fine with me as well. This should be fine as well.

Reply:
I see you have already chanaged it to `MachineConfiguration`. Good then.